Lib/Portability.hpp

```diff
@@ -19,12 +19,6 @@ static_assert(
     "Vampire assumes that there are 8 bits in a `char`"
 );
 
-static_assert(
```

Comment:
Since `TermList` *contains* a pointer, I would say it would be trivially-satisfied. Of course, if you want it for the maintenance value, I'm happy to do it.

Reply:
No, stupid me. Taking it back :)